Cargo.toml

```diff
@@ -4,6 +4,33 @@ members = [
     "integration",
 ]
 
+[workspace.package]
+version = "0.9.0"
```

Comment:
change it to 0.10.0?  we will keep 0.9.x as the "frozen" branch

Reply:
Replace `0.9.0` with `0.10.0` in commit [7a186ef](https://github.com/scroll-tech/scroll-prover/pull/282/commits/7a186ef13ab4fd97850271e4c3b977e96b119ee8).